### ISSUE TITLE
No support for jsonb

### ISSUE THREAD

**Issue by jacobfunch** [balderdashy/sails-postgresql#187]
Is it possible to add support for jsonb and not only json? It's been added to postgresql since 9.4

**Comment by devinivy** [comment]
Will certainly take a PR for that!  How do you propose it be configured in a waterline model?

**Comment by jacobfunch** [comment]
Just as a new type? Or maybe one can take a look at Knex, they define it just like json but with an extra param "true" – which defines it as a jsonb type.

**Comment by sailsbot** [comment]
Thanks for posting, @jacobfunch.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by warpedgeoid** [comment]
Any update on this? Also, why would anyone want a bot to close an issue thread due to inactivity? That makes no sense and only serves to make this issue less visible to commenters.

**Comment by kevinburkeshyp** [comment]
@warpedgeoid the maintainers get a lot of bug-report-y issues that are from beginners, or more suited for stackoverflow, it made it hard to view things over time; I don't blame them. I would suggest reopening this as a new issue.
